ol/tower/src/proof.rs

```diff
@@ -177,33 +173,32 @@ pub fn get_highest_block(blocks_dir: &PathBuf) -> Result<(VDFProof, PathBuf), Er
         }
     }
 
-    if max_block.is_some() && max_block_path.is_some() {
-        return Ok((max_block.unwrap(), max_block_path.unwrap()));
-    } else {
-        bail!(
-            "cannot find a valid VDF proof in files to determine next proof's parameters. Exiting."
-        )
+    if let Some(max_block) = max_block {
+	if let Some(max_block_path) = max_block_path {
+	    return Ok((max_block, max_block_path));
+	}
```

Comment:
small visual issue. this could be indented to look cleaner.

Reply:
Hmm it looks already indented to me, could you clarify which indentation looks cleaner?

Reply:
Awesome! amazing work. Very excited to see you contributing to the codebase